MicrosoftTeams.schema.json

```diff
@@ -450,6 +450,34 @@
                 "type": "string",
                 "maxLength": 2048
             }
+        },
+        "webApplicationInfo": {
+            "type": "object",
+            "properties": {
+                "id": {
+                    "type": "string",
+                    "description": "AAD application id of the app.",
+                    "maxLength": 64
```

Comment:
should we add a format for this to ensure it is a GUID?

Reply:
Good idea; done